docker-compose.yml

```diff
@@ -22,6 +22,31 @@ services:
       - "8080:8080"
       - "8081:8081"
 
+  db:
+    image: postgres:latest
+    networks:
+      - kafka_network
+    environment:
+      POSTGRES_DB: mydb
+      POSTGRES_USER: username
+      POSTGRES_PASSWORD: password
+    volumes:
+      - ./db/data:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
```

Comment:
The postgres database in `zeebe-test-bench` is also using port 5432, can you change it to a different number?

Reply:
The port doesn't need to be exposed since pgAdmin and zeevision are in the same network, so you can remove this line completely.